README.md

```diff
@@ -75,6 +75,43 @@ python -m examples.example1
 
 All APIs and Models' documentation can be found at the [Generated client's README file](kubernetes/README.md)
 
+## Compatibility
+
+`client-python` follows [semver](http://semver.org/), so until the major version of
+client-python gets increased, your code will continue to work with explicitly 
+supported versions of Kubernetes clusters.
+
+#### Compatibility matrix
+
+|                         | Kubernetes 1.3 | Kubernetes 1.4 | Kubernetes 1.5 | Kubernetes 1.6 |
+|-------------------------|----------------|----------------|----------------|----------------|
+| client-python 1.0       | +              | +              | ✓              | -              |
+| client-python 2.0 alpha | +              | +              | +              | ✓              |
+| client-python HEAD      | +              | +              | +              | +              |
+
+Key:
+
+* `✓` Exactly the same features / API objects in both client-python and the Kubernetes
+  version.
+* `+` client-python has features or api objects that may not be present in the
+  Kubernetes cluster, but everything they have in common will work.
+* `-` The Kubernetes cluster has features the client-python library can't use
+  (additional API objects, etc).
+
+See the [CHANGELOG](./CHANGELOG.md) for a detailed description of changes
+between client-python versions.
+
+| Client version | Canonical source for OpenAPI spec    | Maintenance status            |
+|----------------|--------------------------------------|-------------------------------|
+| 1.0 Alpha/Beta | Kubernetes main repo, 1.5 branch     | -                             |
+| 1.0.x          | Kubernetes main repo, 1.5 branch     | ✓                             |
+| 2.0 alpha      | Kubernetes main repo, 1.6 branch     | ✓                             |
+
+Key:
+
+* `✓` Changes in main Kubernetes repo are manually ([should be automated](https://github.com/kubernetes-incubator/client-python/issues/177)) published to client-python when they are available.
+* `-` Deprecated; please upgrade.
```

Comment:
I might word it as "No longer maintained; please upgrade.".  And go with a `✗` rather than a `-`.  Only because sometimes Deprecated things are still maintained for a short time, so it's slightly more clear with the suggested wording.